### DISCUSSION TITLE
Back button

### DISCUSSION THREAD

**Discussion by markandpreston1** [ThioJoe/YT-Spammer-Purge#334] [Ideas]
I am really tired of once i choose a mode, i need to restart the app just to go to the main menu. if its not required, can you add a back or home button? thx

**Comment by TechStudent10** [comment]
I already tried implementing the feature (being able to restart the program) but @ThioJoe closed my PR because he wants to do it a different way.

**Comment by ThioJoe** [comment]
I should be able to add the feature in version 2.10

**Comment by ThioJoe** [comment]
This feature is now available in the latest 2.10.0-Beta1

**Reply by TechStudent10** [reply]
Nice!

**Comment by DinhHuy2010** [comment]
Can you add exit option from main menu?

**Reply by TechStudent10** [reply]
You can just close out the window
